featuretools/tests/primitive_tests/test_transform_primitive.py

```diff
@@ -837,6 +824,7 @@ def test_lag_with_strings():
     primitive_func = primitive_instance.get_function()
 
     array = pd.Series(["hello", "world", "foo", "bar"])
+    array.astype("string")
```

Comment:
This isn't actually changing the type of `array` as you are essentially throwing away the return value from the method call(this is not an in place operation).

You can update the line above to be 
```
array = pd.Series(["hello", "world", "foo", "bar"], dtype="string")
```